packages/cozy-harvest-lib/src/components/KonnectorConfiguration/DataTab/index.jsx

```diff
@@ -61,6 +63,8 @@ export const DataTab = ({
           <LaunchTriggerCard
             konnectorRoot={konnectorRoot}
             flow={flow}
+            intentsApi={intentsApi}
+            account={account}
```

Comment:
Why this addition (of the account)?

Reply:
OK I missed the fact that `LaunchTriggerCard` can be `LaunchTriggerAlert`. 

But you have to change the `LaunchTriggerCard.propTypes` to add the account

Reply:
Also added propTypes for intentsApi property